### ISSUE TITLE
wrong grammar/spelling in german html Output

### ISSUE THREAD

**Issue by andreas-butz** [doxygen/doxygen#9045]
**Describe the bug**
There is a grammar/spelling mistake occuring when generating a german html documentation, in the line documenting the source class of the documentation at the bottom of each html-Documentation-file.
It says "Die Dokumentation für dieseKlasse wurde erzeugt aufgrund der Datei",
but it should be "Die Dokumentation für diese Klasse wurde erzeugt aufgrund der Datei".

**Expected behavior**
The correct sentence should be "Die Dokumentation für diese Klasse wurde erzeugt aufgrund der Datei"
with a blank space between "diese" and "Klasse"

**Screenshots**
current behavior:
![grafik](https://user-images.githubusercontent.com/97887736/149774444-b5e8b2be-d55c-44bf-86ee-47da7d5ff675.png)
expected behavior(manually patched file):
![grafik](https://user-images.githubusercontent.com/97887736/149774811-ee3c1a05-d7b7-4169-945c-00af562a2d17.png)

**To Reproduce**
Generate documentation of an C# class file with 
OUTPUT_LANGUAGE        = German

**Version**
doxygen 1.9.3 on Windows (amd64)

**Additional context**

**Comment by NaincyKumariKnoldus** [comment]
Hey @andreas-butz I would really like to do this. Can I please?

**Comment by albert-github** [comment]
Any help is welcome, but keep in mind that it might also affect other languages and they should be checked as well!

**Comment by andreas-butz** [comment]
> Hey @andreas-butz I would really like to do this. Can I please?

@NaincyKumariKnoldus  I'm new in taking part in the open source community, so i don't know much about the rules.
But i don't think, that anyone must ask anyone, wether it's allowed to help solving a Problem or not...

**Comment by andreas-butz** [comment]
> Hey @andreas-butz I would really like to do this. Can I please?

@NaincyKumariKnoldus I'm new in taking part in the open source community, so i don't know much about the rules.
But i don't think, that anyone must ask anyone, wether it's allowed to help solving a Problem or not...

**Comment by andreas-butz** [comment]
> Any help is welcome, but keep in mind that it might also affect other languages and they should be checked as well!

I've tested every language supported and tryed to translate it with g...le translate
so no guarantee for that:
**not affected by the bug:**
Afrikaans
Arabic
Brazilian (quite sure)
Catalan
Chinese
Chinese-Traditional
Croatian
Czech
Danish (quite sure)
Dutch (quite sure)
English
Esparanto
Farsi
French
Greek
Hungarian
Indonesian
Italian
Japanese-en
Korean
Korean-en
Latvian
Lithuanian
Macedonian
Norwegian
Persian
Polish
Portuguese
Romanian
Russian
Serbian-Cyrillic
Slovak
Slovene
Spanish
Swedish
Turkish
Ukrainian




**affected:**
Finnish: translate says "The documentation for this *category* ..... with "Dokumentaatio tälle luokalle luotiin seuraavasta tiedostosta" and the output for "Dokumentaatio tälle luokalleluotiin seuraavasta tiedostosta" is " The documentation for this *class*...." (just what the translator says, i don't know any Finnish word)

Japanese: translate says: "This *class detail* was extracted from the following file "

Serbian: translate says: "The documentation for this class is based on *a* file"

Vietnamese: translate says: "Information for the class is compiled from the following files:" instead of "file"

German: as described in the first post on this issue

**Comment by albert-github** [comment]
@andreas-butz 
Nice overview.
Problem is that the translation of other languages has to be done by someone who masters that language.
I was hinting at the problem that in other languages a space might be missing as well at the mentioned place like it is in the German translation, doesn't look like to be the case though.

**Comment by albert-github** [comment]
@andreas-butz 
I checked the code for Finnish and this looks (as a layman) OK to me:
```
        case ClassDef::Class:      result+="luokalle"; break; // "class"
  ....
      case ClassDef::Category:   result+="kategorialle"; break; // "category"

```
Pasting:
```
luokalle
kategorialle

Dokumentaatio tälle luokalle luotiin seuraavasta tiedostosta
Dokumentaatio tälle kategorialle luotiin seuraavasta tiedostosta
```
int Google translate gives:
```
class
category

Documentation for this category was created from the following file
Documentation for this category was created from the following file 
```

Might be:
- there is a small bug in a translation in Google translate (as it does some best guesses)
- there are multiple translations possible

As noted before this can only be solved by a person mastering the language (but as a layman the way doxygen handles it looks OK to me)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9050

(The fixed slipped into another proposed pull request, so I now made a separate pull request out of it).

**Comment by andreas-butz** [comment]
> 

@albert-github
Thank you very much
So version 1.9.4 or 1.9.5 will contain the fix?

**Comment by albert-github** [comment]
Fix will be included in the 1.9.4 release.

Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.4.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
